src/page/generate/ViewDocumentPage.tsx

```
@@ -740,6 +643,39 @@ Silakan lanjutkan pembuatan bagian berikutnya dari dokumen tanpa mengulangi kont
+                <Tooltip>
+                  <TooltipTrigger>
+                    <Input key={index} placeholder={part.slice(1, -1)} className="lowercase text-xs h-7 px-2 border text-center rounded-sm max-w-fit py-1 cursor-pointer" />
+                  </TooltipTrigger>
+                  <TooltipContent>
+                    <span className="lowercase text-xs dark cursor-pointer z-[200]">{part.slice(1, -1)}</span>
+                  </TooltipContent>
+                </Tooltip>
+              );
+            }
+            return (
+              <ReactMarkdown remarkPlugins={[remarkGfm]}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)